dev-notes/flashing_rust_sample.md

```diff
@@ -0,0 +1,102 @@
+# Flashing rust hello-world app
+
+## Environment prepare
+
+* Get latest stable Rust version from https://rustup.rs.
+
+* Install required target and tools
+
+```shell
+rustup target add thumbv7em-none-eabihf
+rustup component add llvm-tools-preview
+cargo install cargo-binutils
+cargo install cargo-embed
+cargo install probe-rs-cli
+```
+
+## Building app
+
+This is a custom app based on [blinky-button-demo](https://github.com/nrf-rs/nrf-hal/tree/master/examples/blinky-button-demo)
+sample. Code has been adapted run on nRF52840 dongle.
+
+```shell
+git clone https://github.com/fobnail/nrf-hal
+cd examples/blinky-demo-nrf52840
+cargo build --target thumbv7em-none-eabihf
+```
+
+## Flashing app with J-Link compatible device
+
+This is the simpler procedure, it requires J-Link (or compatible device) or
+CMSIS-DAP device. I am using nRF52840-DK development board (J-Link compatible).
```

Comment:
@arturkow2000 some link to the board would be nice

Reply:
done